nvflare/private/fed/server/admin.py

```diff
@@ -222,7 +150,7 @@ def __init__(
         self.file_upload_dir = file_upload_dir
         self.file_download_dir = file_download_dir
 
-        # YC: need to register FileTransferModule any more
+        # YC: no need to register FileTransferModule any more
```

Comment:
since you change this message, should we just remove the whole commented out code?

Reply:
Removed.